### DISCUSSION TITLE
github student developer verification done and was approved but it is more than 3 days still I can't access the student benefits.

### DISCUSSION THREAD

**Discussion by dagdiyap** [community/community#138010] [GitHub Education]
Can someone please help me with it.

**Comment by Shriharsh-Deshmukh** [comment]
Hello @dagdiyap,
I also had a similar experience. After being approved, I received the student benefits within a day. It didn't take long for them to reflect on my profile— you'll notice a pro icon appear, indicating that your benefits are active. If you haven't seen any changes yet, it might just be a short delay, but they should come through soon!

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @dagdiyap ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by joemar25** [comment]
If your GitHub Student Developer Pack was approved but you still can't access the benefits after 3 days, try the following:

1. **Log out and back in** to your GitHub account to refresh your session.
2. **Clear your browser cache** and cookies.
3. **Check your email** for any further instructions or confirmation messages.
4. **Verify your account status** on the GitHub Education page to make sure it's fully activated.
5. If none of these work, **contact GitHub support** for assistance.

**Comment by gauravbhindwar** [comment]
github student developer verification done and was approved but it is more than 3 days still I can't access the student benefits
